src/pages/Transaction/TransactionCellScript/index.tsx

```diff
@@ -106,7 +106,7 @@ const handleFetchScript = (
         setScriptFetchStatus(dispatch, true)
         let dataValue: State.Data
         if (cell.cellInfo.data !== '0x') {
-          dataValue = { data: hexToUtf8(cell.cellInfo.data.substr(2)) }
+          dataValue = { data: cell.cellInfo.data }
```

Comment:
I think you can reuse lines 80 to 84 of the code

Reply:
Data structure and logic are different.